docs/framework/configure-apps/file-schema/windows-identity-foundation/add.md

```diff
@@ -7,10 +7,11 @@ author: "BrucePerlerMS"
 # \<add>
 Adds the specified security token handler to the token handler collection.  
   
- \<system.identityModel>  
-\<identityConfiguration>  
-\<securityTokenHandlers>  
-\<add>  
+[**\<configuration>**](../configuration-element.md)\
```

Comment:
Shouldn't we just put these in xml code blocks? @mairaw

Reply:
they link to the specific elements, so we can't put them in xml code blocks

Reply:
Let's debate on this again? 😁

Reply:
Let's debate on this again? 😁